yarn-project/noir-compiler/README.md

```diff
@@ -1,15 +1,12 @@
 # Aztec Noir compiler
 
-The Aztec noir compiler compiles noir contracts using noir_wasm and outputs Aztec formatted contract ABIs.
+The Aztec noir compiler compiles noir contracts using nargo or noir_wasm and outputs Aztec formatted contract ABIs. The compiler can also generate typescript classes for each contract, as well as Noir interfaces for calling external functions.
 
 ## Installation
 
 To install the package, just run `yarn add @aztec/noir-compiler`.
 
 ## Usage
 
-To run the compiler as a CLI tool, run `yarn aztec_noir_compiler compile <path_to_noir_contract_crate>`
+To run the compiler as a CLI tool, first install it and then run: `yarn aztec-compile compile --help`
```

Comment:
"..., first install the aztec-cli: `npm install -g @aztec/cli` and then run..."?

Reply:
It can be used from both packages! So if you don't want the full-blown aztec cli and just want to compile, you can install the noir compiler only. I'd expect most users to just go via the aztec cli, but it's nice to have a more lightweight alternative. I've added a clarification for this here in the README.